templates/home/index.html.twig

```diff
@@ -1,7 +1,59 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Student{% endblock %}
+
+    {% block body %}
+        <ul class="list-group m-4">
+            <li class="list-group-item list-group-item-info">Studentai</li>
+            {% for student in someVariable %}
+                <li class="list-group-item">
+                    <a href="{{ path('student') }}?name={{ student.student|url_encode }}&project={{ student.project|url_encode }}">{{ student.student|escape }}</a>
+                    ( <span class="badge">  Mentorius</span> {{ student.mentor|escape }})
```

Comment:
Kad nereikėtų rašyti visiems to paties. žr. testų rašytus komentarus:
![image](https://user-images.githubusercontent.com/1453957/69900983-7acb2a80-1383-11ea-8b69-aa9d1f08d6be.png)
per https://github.com/aurelijusb/kickstart/pull/148/files